scripts/deploy/deploy.sh

```diff
@@ -0,0 +1,35 @@
+if [ "$TRAVIS_PULL_REQUEST" != "false" -o "$TRAVIS_BRANCH" != "staging" -o "$TRAVIS_BRANCH" != "production" ]; then
+    echo "Skipping deploy to staging or production server; The request or commit is not on staging or production branch"
+    exit 0
+fi
+
+if [ "$TRAVIS_PULL_REQUEST" != "false" -a "TRAVIS_BRANCH" != "staging"]; then
```

Comment:
Please modify line 6-30 and put them in a function(). The logic could be something like this

```
if branch == staging:
    deploy to evalai-staging-* ECR repos with staging credentials
else if branch == production:
   deploy to evalai-* ECR repos with prod credentials
```